content/docs/2.10/scalers/github-runner.md

```diff
@@ -0,0 +1,153 @@
+      labels: "{labels}"
+      # Optional: The target number of queued jobs to scale on
+      targetWorkflowQueueLength: "1" # Default 1
+      # Optional: The target number of queued jobs to achieve to activate the scaler
+      activationTargetWorkflowQueueLength: "0" # Default 0
+    authenticationRef:
+      name: personalAccessToken
+```
+
+**Parameter list:**
+
+- `githubAPIURL` - The URL of the GitHub API, defaults to https://api.github.com. You should only need to modify this if you have your own GitHub Appliance. (Optional)
+- `owner` - The owner of the GitHub repository, or the organization that owns the repository. (Required)
+- `runnerScope` - The scope of the runner, can be either "org", "ent" or "repo". (Required)
+- `personalAccessTokenFromEnv` - The personal access token to access the GitHub API. (Required)
+- `repos` - The list of repositories to scale, separated by comma. (Optional)
+- `labels` - The list of runner labels to scale on, separated by comma. (Optional)
+- `targetWorkflowQueueLength` - The target number of queued jobs to scale on. (Optional, Default: 1)
+- `activationTargetWorkflowQueueLength` - The target number of queued jobs to achieve to activate the scaler. (Optional, Default: 0)
+
+*Parameters from Environment Variables*
+
+You can access each parameter from above using environment variables. When you specify the parameter in metadata with a suffix of `FromEnv`, 
+the scaler will use the value from the environment variable. The environment variable must be available to the manifest. e.g. `labelsFromEnv: "RUNNER_LABELS"` will use the environment variable `RUNNER_LABELS` as the source fo the `labels` parameter.
+
+There are also a set of default environment variables that will be used if none of the above are specified directly.
+
+- `githubAPIURL` - `GITHUB_API_URL`
+- `runnerScope` - `RUNNER_SCOPE`
+- `personalAccessTokenFromEnv` - `ACCESS_TOKEN`
+- `labels` - `LABELS`
+- `owner` - No defaults, must be specified if runnerScope is not `org` or `ent`. Otherwise `ORG_NAME` or `ENTERPRISE_NAME` will be used.
+- `repos` - No defaults, must be specified or left empty
+- `targetWorkflowQueueLength` - No defaults, must be specified or left empty
+- `activationTargetWorkflowQueueLength` - No defaults, must be specified or left empty
+
+### Authentication Parameters
+
+As an alternative to using environment variables, you can authenticate with GitHub using a Personal Access Token via `TriggerAuthentication` configuration.
+
+**Personal Access Token Authentication:**
+
+- `personalAccessToken` - The Personal Access Token (PAT) for GitHub from your user.
+
+### How does it work?
+
+The scaler will query the GitHub API to get the number of queued jobs in the specified repositories. If the number of queued jobs is greater than the `targetWorkflowQueueLength`, the scaler will scale up. If the number of queued jobs is less than the `activationTargetWorkflowQueueLength`, the scaler will scale down.
```

Comment:
Do we need 2 configuration values here? For typical queue-based workloads we only have 1.

Reply:
activation removed